bundles/org.openhab.binding.lifx/README.md

```diff
@@ -19,7 +19,7 @@ The following table lists the thing types of the supported LIFX devices:
 | LIFX Downlight               | colorlight   |
 | LIFX GU10                    | colorlight   |
 | LIFX Mini Color              | colorlight   |
-| LIFX Tile                    | colorlight   |
+| LIFX Tile                    | tile         |
```

Comment:
It would be more consistent with the other thing types to name this a `tilelight`. It will also help to distinguish between LIFX products if they start selling other (non-lighting) products similar to Philips Hue & IKEA Tradfri.

```suggestion
| LIFX Tile                    | tilelight    |
```

Reply:
I just read they are working on a [LIFX wall switch](https://twitter.com/davemclauchlan/status/1186502303647334401?s=21). :wink: